lib/orchestrator/retrieve_next_job.rb

```diff
@@ -7,7 +7,7 @@ def call
       locked_key = Exercism::ToolingJob.key_for_locked
 
       job_id = redis.lpop(queued_key)
-      return unless job_id
+      return unless job_id && job_id != ""
```

Comment:
No `blank?` available here :(

Reply:
There's `empty?` actually. I'll change it to that :)